Author: Philipp Benner <mail@philipp-benner.de>
Description: `\fB' and `\fP' are not appropriate for the userinput environment.
Instead, `.ft' is used as a replacement. See bug #211466 for
discussion.
--- a/Transpec/docbook-to-man.ts
+++ b/Transpec/docbook-to-man.ts
@@ -1930,8 +1930,8 @@
 -
 #
 GI:		USERINPUT
-StartText:	\\fB
-EndText:	\\fP
+StartText:	^.ft\sB^
+EndText:	^.ft
 -
 #
 GI:		VARNAME
